libs/converter-nh-accuvote/README.md

```diff
@@ -7,33 +7,36 @@ VotingWorks election definition.
 
 Follow the instructions in the [VxSuite README](../../README.md) to get set up.
 
-## API Usage
-
-```ts
-import { convertElectionDefinition } from '@votingworks/converter-nh-accuvote';
-import { loadImageData } from '@votingworks/image-utils';
-import { DOMParser } from '@xmldom/xmldom';
-
-const front = await loadImageData('./template-front.jpeg');
-const back = await loadImageData('./template-back.jpeg');
-const definition = new DOMParser().parseFromString(
-  await fs.readFile('./election.xml', 'utf8'),
-  'text/xml'
-);
-const convertResult = convertElectionDefinition({
-  front,
-  back,
-  definition,
-});
-
-if (convertResult.isErr()) {
-  console.error(`error: ${JSON.stringify(convertResult.err())}`);
-} else {
-  console.log('Converted election result:', convertResult.ok().election);
+## CLI Usage
+
+### Correct an AccuVote XML definition
```

Comment:
Do you foresee this always being a separate step? Or might we integrate it into the main command?